tests/projects/test_projects.py

```diff
@@ -199,15 +199,15 @@ def test_run_local_git_repo(local_git_repo,
     submitted_run.wait()
     validate_exit_status(submitted_run.get_status(), RunStatus.FINISHED)
     # Validate run contents in the FileStore
-    run_id = submitted_run.run_id
+    run_uuid = submitted_run.run_id
```

Comment:
i think these are all meant to be `run_id` from aaron's PR?

Reply:
Good catch! Broken merge. I'll fix these.

Reply:
Actually I wanted to revert this whole file since I decided projects were out of scope of this PR..